src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState('');
+  // eslint-disable-next-line
+  const URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+  const loadPeople = async () => {
+    try {
+      const response = await fetch(URL);
+
+      if (!response.ok) {
+        throw new Error(`HTTP error! Status: ${response.status}`);
+      }
+
+      const peopleData = await response.json();
```

Comment:
Would be nice to move that code to api file (if I am not mistaken that file was presented in the task source code)